### PR TITLE
Fix examples for r2022b release

### PR DESCRIPTION
**Pull request by ygoumaz**
Examples from the package had a few warnings or errors due to modifications done for Webots R2022b:

- [x] **catch_the_bird**: the rangeFinder name is containing a "minus" symbol, which is not supported as graph resource name.
- [x] **complete_test**: the robot window is now opening in the web browser. The HTML file needs to be updated.
- [x] **e_puck_line**: Currently it is not very clear to the user why the simulation suddenly stops working when the epuck bumps into an obstacle. This PR adds a warning telling the user to complete the missing parts of the code or to remove the obstacle.
- [x] **panoramic_view_recorder**: Update the controller from "void" to "<generic>".
- [x] **tiago**: A local texture is missing. Use a PROTO appearance instead.

In addition, all worlds should be converted to EXTERNPROTO, but this will be addressed in another PR, as this is non-blocking to continue R2022b release tests (#116).

### CODE REVIEW

File: src/catch_the_bird.cpp [hunk 1]
```diff
@@ -79,6 +79,7 @@ int main(int argc, char **argv) {
     ROS_ERROR("Failed to call service device_list.");
   motorName = deviceList[0];
   rangeFinderName = deviceList[1];
+  std::replace(rangeFinderName.begin(), rangeFinderName.end(), '-', '_');
```
Comment (omichel) [inline src/catch_the_bird.cpp:82]: Isn't it simpler (and better) to rename this device in the world file instead?

File: src/e_puck_line.cpp [hunk 2]
```diff
@@ -609,6 +609,16 @@ int main(int argc, char **argv) {
     leftWheelVelocityClient.call(wheelSrv);
     wheelSrv.request.value = (M_PI / 1000.0) * speed[RIGHT];
     rightWheelVelocityClient.call(wheelSrv);
+
+    if (oam_active) {
+      ROS_WARN("The robot has bumped into an obstacle. Please complete the code in the \"LineLeavingModule()\" function of "
+               "\"e_puck_line.cpp\" or simply remove the obstacle to get a correct behavior.");
```
Comment (omichel) [inline src/e_puck_line.cpp:615]:
```suggestion
               "\"%s\" or simply remove the obstacle to get a correct behavior.", __FILE__);
```

File: worlds/catch_the_bird.wbt [hunk 3]
```diff
@@ -28,6 +28,7 @@ Robot {
         rotation 1 0 0 1.5708
         children [
           RangeFinder {
+            name "range_finder" # Valid characters are a-z, A-Z, 0-9, / and _
```
Comment (omichel) [inline worlds/catch_the_bird.wbt:31]:
```suggestion
            name "range_finder"  # valid characters for ROS service/topic names are a-z, A-Z, 0-9, / and _ 
```

**Review by omichel — CHANGES_REQUESTED** [review]

File: worlds/catch_the_bird.wbt (hunk 3 above)
Comment (ygoumaz) [reply worlds/catch_the_bird.wbt:31]: Much more complete indeed.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**